### DISCUSSION TITLE
PaperSize Usage example for docs

### DISCUSSION THREAD

**Discussion by SandroPanchame** [py-pdf/pypdf#1545] [General]
Hello, 

I've been working on the documentation for the PaperSize class, and I'm wondering if what I've currently done is okay to submit for a pull request.

Discussion for adding example usage was discussed here: 

https://github.com/py-pdf/pypdf/pull/1447 

An example of the page is here:

[The PaperSize Class — pypdf documentation.pdf](https://github.com/py-pdf/pypdf/files/10396287/The.PaperSize.Class.pypdf.documentation.pdf)

Furthermore, I'm not to sure about what I currently have, and I feel like it could be trimmed down a bit more. The docstring wasn't extended, and the usage examples were added to the .rst file for the PaperSize class.

I'd greatly appreciate some input on the doc.

**Comment by MartinThoma** [comment]
That looks good!

The paper size class is not complicated, so it's fine if there is not a lot. The examples add value. Thank you :-) 

(small adjustment: I would write the first letter in headings with a capital letter)
